### PR TITLE
Update commands.md

### PR DESCRIPTION
**Pull request by LiorKogan**

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -248,13 +248,13 @@ If this command is used to add data to an existing timeseries, `retentionTime` a
 Create a compaction rule.
 
 ```sql
-TS.CREATERULE sourceKey destKey AGGREGATION aggregationType timeBucket
+TS.CREATERULE sourceKey destKey AGGREGATION aggregationType bucketDuration
```
Comment (OfirMos) [inline docs/commands.md:251]: @LiorKogan  We need to change the code to support this change right? (CC: @danni-m )

**Review by OfirMos — COMMENTED** [review]
Need a code change to support all this

File: docs/commands.md (hunk 1 above)
Comment (LiorKogan) [reply docs/commands.md:251]: No, I just changed the name of the parameter, to make it more clear....

**Comment by danni-m** [comment]
@OfirMos @LiorKogan we need to rebase this and fix conflicts.

**Pull request closed without merging**